src/shared/components/VideoPreviewBase/VideoPreviewBase.tsx

```diff
@@ -197,17 +274,43 @@ const VideoPreviewBase: React.FC<VideoPreviewBaseProps> = ({
                   </Anchor>
                 ))}
               {showMeta && (
-                <MetaContainer main={main}>
+                <MetaContainer noMarginTop={showChannel === false} main={main}>
```

Comment:
```suggestion
                <MetaContainer noMarginTop={!showChannel} main={main}>
```

Shouldn't really be a problem in this case, but in general this should be a safer construct (for example to handle `undefined` as well)